### ISSUE TITLE
Ambiguity in Loss function

### ISSUE THREAD

**Issue by nazib** [balakg/voxelmorph#4]
Hi there
I tried to train your network with 663 MRI images from ABIDE dataset. Before applying to the network I change the data dimension to 256x256x256 and changed the network dimension respectively.  I randomly picked one image from the dataset(ABIDE in our case) as atlas and tried to align all other images to it. I tried two runs, one with lamda=1.0 (image 2) and another with lambda=1.5(image 1). 

Checking the loss functions It seems network is not learning anything. Is it because of small number training images or the changed network dimension? It would be helpful for us if you upload some training loss plots if possible.

regards

![lambda1 5](https://user-images.githubusercontent.com/14937060/39907652-6ca1e7c0-552d-11e8-8669-0e4fc335e11d.png)

![lambda1](https://user-images.githubusercontent.com/14937060/39907635-58bfd910-552d-11e8-9cb0-a6901bbe7b79.png)

**Comment by nazib** [comment]
Also do i need to normalize the intensity range of data in 0 to 1 before applying into the network? I just have noticed that the intensity range found in the given altal_norm.npz file is 0 to 0.7276.

**Comment by adalca** [comment]
Both the changed dimensions and the number of images (~600) should be fine. 

However, yes, they should be normalized to roughly 0-1 -- the loss weights reported in the paper assume this normalization. Could you try this and get back to us?

**Comment by nazib** [comment]
Hi Thanks for quick reply. Yep its working now for normalized images. 
![loss](https://user-images.githubusercontent.com/14937060/40032358-2ead1bc6-5837-11e8-95d7-e3662e24e010.png)

**Comment by adalca** [comment]
Great! Glad we could help.
